pages/common/newsboat.md

```diff
@@ -0,0 +1,16 @@
+# newsboat
+
+> An RSS/Atom feed reader for the text console.
+> More information: https://newsboat.org/.
+
+- First import feed urls from an OPML file
+`newsboat -i {{my-feeds.xml}}
+
+- Alternatively, add feeds manually
+`echo {{http://rss.cnn.com/rss/cnn_topstories.rss}} >> $HOME/.newsboat/urls`
```

Comment:
Perhaps this will read better?
```suggestion
- Alternatively, add feeds manually:

`echo {{http://example.com/path/to/feed}} >> "${HOME}/.newsboat/urls"`
```